### ISSUE TITLE
Disable liquid as a feature still requires the crate

### ISSUE THREAD

**Issue by sylvestre** [sharkdp/bat#1159]
Trying to update bat to a more recent version in Debian and Ubuntu, I had to try to disable the "liquid" feature (liquid has been uploaded in Debian but the review process can take months...)

My patch is something like
```
Index: bat/Cargo.toml
===================================================================
--- bat.orig/Cargo.toml
+++ bat/Cargo.toml
@@ -109,12 +109,12 @@ version = "0.3"
 version = "2.33"
 optional = true
 
-[build-dependencies.liquid]
-version = "0.21"
-optional = true
+#[build-dependencies.liquid]
+#version = "0.21"
+#optional = true
 
 [features]
-application = ["atty", "clap", "dirs", "git", "lazy_static", "liquid", "paging", "wild"]
+application = ["atty", "clap", "dirs", "git", "lazy_static", "paging", "wild"]
 default = ["application"]
 git = ["git2"]
 paging = ["shell-words"]
```

but the build fails with:
```
error[E0433]: failed to resolve: use of undeclared type or module `liquid`
  --> build.rs:34:21
   |
34 |     let variables = liquid::object!({
   |                     ^^^^^^ use of undeclared type or module `liquid`

error[E0433]: failed to resolve: use of undeclared type or module `liquid`
  --> build.rs:22:21
   |
22 |         variables: &liquid::Object,
   |                     ^^^^^^ use of undeclared type or module `liquid`

error[E0433]: failed to resolve: use of undeclared type or module `liquid`
  --> build.rs:26:24
   |
26 |         let template = liquid::ParserBuilder::with_stdlib()
   |                        ^^^^^^ use of undeclared type or module `liquid`

error: aborting due to 3 previous errors
```

Either it is a mandatory dep or builds.rs should handle the case when liquid is disabled.

**Comment by sharkdp** [comment]
liquid *is* an optional *build* dependency for "bat as a library":
https://github.com/sharkdp/bat/blob/803447e2a1a6967f09cd7962183facdabc2b7a13/build.rs#L4-L7

It is not optional for "bat as an application", unfortunately.

If it's a huge problem to package it for Debian, we could think about removing the dependency. It was a bit overkill to use a template engine anyway.

@eth-p FYI

**Comment by sylvestre** [comment]
Yeah, liquid isn't a trivial piece of packaging and it is going to take a while to get it into the Debian & Ubuntu archive.
So, if you could it would be great (or just the patch to disable it)
thanks!

**Comment by sylvestre** [comment]
Currently, bat will not ship in Debian Bulleyes, this is why I have been trying to fix that!

**Comment by eth-p** [comment]
If this is stopping bat from making it into distro repositories, we should just replace the template engine with find and replaces.

I used liquid to futureproof document templating, but it's not worth it when it actively works against user and distro adoption.

**Comment by sharkdp** [comment]
Fixed via f18009e5d57bb4ee03511a9a2b6fd933d58d539f.

**Comment by sylvestre** [comment]
@sharkdp Are you planning a new release soon? (to know if I should work on a backport of the patch or not)

**Comment by sharkdp** [comment]
I'll try to publish a new release soon. Are there any deadlines for Debian/Ubuntu packaging?

**Comment by SuperSandro2000** [comment]
The package is non essential so you have a bit more time. I am unsure how much but probably until 2021 for Debian.

https://release.debian.org/bullseye/freeze_policy.html

Ubuntu 20.10 Groovy is already frozen https://lists.ubuntu.com/archives/ubuntu-devel-announce/2020-August/001279.html

**Comment by sharkdp** [comment]
I have published a new release, v0.16.0
